sdks/python/apache_beam/yaml/standard_io.yaml

```
@@ -257,3 +257,29 @@
         'WriteToJdbc': 'beam:schematransform:org.apache.beam:jdbc_write:v1'
       config:
         gradle_target: 'sdks:java:extensions:schemaio-expansion-service:shadowJar'
+
+- type: renaming
+  transforms:
+    'ReadFromSpanner': 'ReadFromSpanner'
+    'WriteToSpanner': 'WriteToSpanner'
+  config:
+    mappings:
+      'ReadFromSpanner':
+        project_id: 'project_id'
+        instance_id: 'instance_id'
+        database_id: 'database_id'
+        table_id: 'table_id'
+        query: 'query'
+        columns: 'columns'
+      'WriteToSpanner':
+        instance_id: 'instance_id'
+        database_id: 'database_id'
+        table_id: 'table_id'
```

I would strip the `_id` from all of these parameters on the YAML side

i.e.
```suggestion
        project: 'project_id'
        instance: 'instance_id'
        database: 'database_id'
        table: 'table_id'
        query: 'query'
        columns: 'columns'
      'WriteToSpanner':
        instance: 'instance_id'
        database: 'database_id'
        table: 'table_id'
```